tests/pipelines/controlnet/test_controlnet.py

```diff
@@ -460,6 +460,26 @@ def test_save_pretrained_raise_not_implemented_exception(self):
             except NotImplementedError:
                 pass
 
+    def test_inference_nested_image_input(self):
+        device = "cpu"
+
+        components = self.get_dummy_components()
+        sd_pipe = StableDiffusionControlNetPipeline(**components)
+        sd_pipe = sd_pipe.to(torch_device)
+        sd_pipe.set_progress_bar_config(disable=None)
+
+        inputs = self.get_dummy_inputs(device)
+        inputs["prompt"] = [inputs["prompt"], inputs["prompt"]]
+        inputs["image"] = [inputs["image"], inputs["image"]]
```

Comment:
this is not list of list though, no? it is just a list of 2 images?

Reply:
https://github.com/huggingface/diffusers/blob/79df50388df09d9615e3c067695a453bb0a694c0/tests/pipelines/controlnet/test_controlnet.py#L381-L392

`inputs["image"]` is a list of image, so here comes a list of image list I think.